components/perl/Tk/Makefile

```diff
@@ -0,0 +1,63 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2021, Friedrich Kink. All rights reserved.
+#
+
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		Tk
+COMPONENT_VERSION=	804.036
+IPS_COMPONENT_VERSION=	5.11
+COMPONENT_REVISION=	1
```

Comment:
The IPS_COMPONENT_VERSION should be 804.0.36 but not 5.11. This version number is reserved for packages tightly coupled with illumos gate.
Also please add
HUMAN_VERSION= $(COMPONENT_VERSION) and also add an appropriate entry in the manifest. This will add information will that can be used outside IPS (eg, by humans or repology).